sql/identities/identities.go

```diff
@@ -2,15 +2,38 @@ package identities
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"time"
 
 	sqlite "github.com/go-llsqlite/crawshaw"
+	"go.uber.org/zap"
 
 	"github.com/spacemeshos/go-spacemesh/common/types"
 	"github.com/spacemeshos/go-spacemesh/sql"
 )
 
+const (
+	// Bloom filter size is < 234 KiB while below 100k identities.
```

Comment:
Do you mean identities in general or malicious ones?

```suggestion
	// Bloom filter size is < 234 KiB while below 100k malicious identities.
```